dev/test-anaconda-compatibility-in-docker.sh

```diff
@@ -10,11 +10,13 @@ set -euo pipefail
 
 . ~/.bashrc
 
-pip freeze > /tmp/before.txt
+IGNORE_PATTERN="^\(importlib-metadata\|zipp\)"
```

Comment:
Added this line avoid a failure caused by adding `importlib_metadata >= 3.7.0` as a dependency.

Reply:
What's the failure? Is this going to cause disruptions for real users / use cases?